pastis/launchers/run_luvex_efields.py

```diff
@@ -48,13 +49,21 @@
     # Calculate the static tolerances
     c_target = 6.3*1e-11
     mus = calculate_segment_constraints(pastis_matrix, c_target=c_target, coronagraph_floor=contrast_floor)
-    np.savetxt(os.path.join(dir_run, 'mus_%.2e_%d.csv' % (c_target, NUM_RINGS)), mus, delimiter=',')
+    np.savetxt(os.path.join(dir_run, f'mus_{c_target:.2e}_{NUM_RINGS:d}.csv'), mus, delimiter=',')
```

Comment:
This was just a cosmetic change, matter of personal preference.

Reply:
f-strings are easier to wrote, easier to read, and faster:
https://realpython.com/python-f-strings/